admin_guide/garbage_collection.adoc

```diff
@@ -0,0 +1,100 @@
+= Garbage Collection
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+The OpenShift node performs two types of garbage collection:
+
+* link:#container-garbage-collection[Container Garbage Collection]:
+Removes terminated containers. Typically run every minute.
+* link:#image-garbage-collection[Image Garbage Collection]:
+Removes images not referenced by any running pods. Typically run every 5 minutes.
+
+[[container-garbage-collection]]
+
+=== Container Garbage Collection
+
+The policy for container garbage collection is based on three settings:
+
+* `minimum-container-ttl-duration` - The minimum age that a container is eligible for garbage collection.
+  The default is 1m (one minute). Use 0 for no limit. Values for this setting can be specified using
+  unit suffixes such as h for hour, m for minutes, s for seconds.
+* `maximum-dead-containers-per-container` - The number of instances to retain per pod container.
+  The default is 2.
+* `maximum-dead-containers` - The maximum number of total dead containers in the node.
+  The default is 100.
+
+The `maximum-dead-containers` setting will take precedence over the `maximum-dead-containers-per-container` setting
+when there is a conflict. For example, if retaining the number of `maximum-dead-containers-per-container` would result in a
+total number of containers that is greater than `maximum-dead-containers`. The oldest containers will be removed to satisfy
+the `maximum-dead-containers` limit.
+
+When the node removes the dead containers, all files inside those containers are removed as well. Only containers created
+by the node will be garbage collected.
+
+You can specify values for these settings in the `kubeletArguments` section of the
+link:master_node_configuration.html#node-configuration-files[node configuration file] (node-config.yaml):
+
+====
+.Container Garbage Collection Settings in Node configuration
+[source,yaml]
+----
+
+kubeletArguments:
+  minimum-container-ttl-duration:
+    - 10s
+  maximum-dead-containers-per-container:
+    - 2
+  maximum-dead-containers:
+    - 100
+----
+====
+
+
+[[image-garbage-collection]]
+
+=== Image Garbage Collection
+
+Image garbage collection relies on disk usage as reported by cAdvisor on the node to decide
+which images to remove from the node. It takes the following settings into consideration:
```

Comment:
Can we make it a bit clearer that image GC occurs when you're running out of free space for images?